### ISSUE TITLE
CLI docs: cloud-init devel hotplug-hook <subcommand> --help is empty  or lacks mention of required --subsystem param.

### ISSUE THREAD

**Issue by blackboxsw** [canonical/cloud-init#4827]
# Bug report
No parameter information about the required --subsystem option is provided by `cloud-init devel hotplug-hook query --help` or `handle --help`

```
root@test-n:~# cloud-init devel hotplug-hook  query -h
usage: /usr/bin/cloud-init devel hotplug-hook query [-h]

options:
  -h, --help  show this help message and exit

root@test-n:~# cloud-init devel hotplug-hook  handle --help
usage: /usr/bin/cloud-init devel hotplug-hook handle [-h] -d PATH -u {add,remove}

options:
  -h, --help            show this help message and exit
  -d PATH, --devpath PATH
                        Sysfs path to hotplugged device
  -u {add,remove}, --udevaction {add,remove}
                        Specify action to take.
```
Expected representation of the required --subsystem parameter in both cases.

Affects cloud-init 21b2b6e4423b0fec325b32042c05ef4274cdd301 and earlier

**Comment by holmanb** [comment]
> Affects cloud-init https://github.com/canonical/cloud-init/commit/21b2b6e4423b0fec325b32042c05ef4274cdd301 and earlier

@blackboxsw are you saying that this was introduced in the dhcpcd PR? That is surprising, since that PR doesn't touch any files in the `cloudinit/cmd/` subdirectory.

**Comment by blackboxsw** [comment]
> > Affects cloud-init [21b2b6e](https://github.com/canonical/cloud-init/commit/21b2b6e4423b0fec325b32042c05ef4274cdd301) and earlier
> 
> @blackboxsw are you saying that this was introduced in the dhcpcd PR? That is surprising, since that PR doesn't touch any files in the `cloudinit/cmd/` subdirectory.

No, I was just representing that this bug still affects tip of main at the commit https://github.com/canonical/cloud-init/commit/21b2b6e4423b0fec325b32042c05ef4274cdd301.   This is probably confusing to track a commit in upstream main. We can redact the commit hash if you think that's preferable because any live bug should be considered something as something that affects tip of main.
